scripts/update_versions.sh

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env bash
 
+python3 -m venv venv
```

Comment:
I guess this depends on where this runs, but if there is a possibility that `venv` already exists, then maybe add a check for that directory. Only create the venv virtualenv if it doesn't already exist

Reply:
If it already exists, the command is a noop.

Reply:
Oh thats nice ok 👍